docs/api/modules/Internationalization.md

```diff
@@ -0,0 +1,196 @@
+* [Loading Language Packs](#loading-language-packs)
+  - * [`loadLanguagePack`](#`loadLanguagePack`)
+  - * [`queryLanguagePack`](#`queryLanguagePack`)
+  - * [`updateLocale`](#`updateLocale`)
+  - * [`getLocalePack`](#`getLocalePack`)
+
+
+## Locale Structure
+
+Following the below structure will allow [one-app-locale-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-locale-bundler) to correctly
+bundle your locale files.
+
+Module Root:
+```
+module
+└── locale
+| └── en-US.json
+| └── es-MX.json
+|── src
+|   └── index.js
+└── package.json
+```
+
+This will result in:
+```
+build
+└── 1.0.0
+    ├── <moduleName>.browser.js
+    ├── <moduleName>.legacy.browser.js
+    ├── <moduleName>.node.js
+    └── en-us
+    |    ├── <moduleName>.json
+    |    ├── integration.json
+    |    └── qa.json
+    └──es-MX
+        ├── <moduleName>.json
+        ├── integration.json
+        └── qa.json
+```
+
+The language pack itself needs to be a flat JSON object and will not work with a nested structure.
+
+Example `en-US.json`:
+```JSON
+{
+  "loading": "Loading...",
+  "welcome": "welcome to my module"
+}
+```
+
+### Environment Specific Data
+
+[one-app-locale-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-locale-bundler) also allows your module to provide environment
+specific data. Currently One App supports three different environments;
+production, qa and integration (development).
+
+Below is an example of how you could include link specific data.
+
+Module Root:
+```
+module
+└── locale
+| └── en-US
+|     └── links
+|     |   └── production.json
+|     |   └── qa.json
+|     |   └── integration.json
+|     └── copy.json
+| └── es-MX.json
+|── src
+|   └── index.js
+└── package.json
+```
+
+Example `en-US/copy.json`:
+```JSON
+{
+  "loading": "Loading...",
+  "welcome": "welcome to my module"
+}
+```
+
+Example `en-US/links/production.json`:
+```JSON
+{
+  "example": "https://example.com/en-us/home"
+}
+```
+
+Example `en-US/links/qa.json`:
+```JSON
+{
+  "example": "https://qa.example.com/en-us/home"
+}
+```
+
+Example `en-US/links/integration.json`:
+```JSON
+{
+  "example": "localhost:8080/en-us/home"
+}
+```
+
+This will result in:
+```
+build
+└── 1.0.0
+    ├── <moduleName>.browser.js
+    ├── <moduleName>.legacy.browser.js
+    ├── <moduleName>.node.js
+    └── en-us
+    |    ├── <moduleName>.json
+    |    ├── integration.json
+    |    └── qa.json
+    └──es-MX
+        ├── <moduleName>.json
+        ├── integration.json
+        └── qa.json
+```
+
+Example `en-US/qa.json`:
+```JSON
+{
+  "loading": "Loading...",
+  "welcome": "welcome to my module",
+  "links": {
+    "example": "https://qa.example.com/en-us/home"
+  }
+}
+```
+
+### `ONE_CLIENT_LOCALE_FILENAME`
+
+One App uses the environment variable `ONE_CLIENT_LOCALE_FILENAME` to
+determine which language pack will be loaded. `ONE_CLIENT_LOCALE_FILENAME`
```

Comment:
I am not sure the connection between this and the previous section is as clear as it could be. Maybe we add `...which language pack will be loaded *if using the environment specific language pack feature as described in the previous section*.`?